diff --git a/src/lua_state.cpp b/src/lua_state.cpp
--- a/src/lua_state.cpp
+++ b/src/lua_state.cpp
@@ -192,7 +192,6 @@ void LuaState::_bind_methods()
     ClassDB::bind_method(D_METHOD("to_integer", "index"), &LuaState::to_integer);
     ClassDB::bind_method(D_METHOD("to_vector3", "index"), &LuaState::to_vector3);
     ClassDB::bind_method(D_METHOD("to_boolean", "index"), &LuaState::to_boolean);
-    ClassDB::bind_method(D_METHOD("to_string", "index"), &LuaState::to_string);
     ClassDB::bind_method(D_METHOD("to_string_inplace", "index"), &LuaState::to_string_inplace);
     ClassDB::bind_method(D_METHOD("to_string_name", "index"), &LuaState::to_string_name);
     ClassDB::bind_method(D_METHOD("get_namecall"), &LuaState::get_namecall);
